charts/aws-mountpoint-s3-csi-driver/templates/node.yaml

```diff
@@ -155,7 +155,8 @@ spec:
               mountPropagation: HostToContainer
               {{- end }}
             - name: systemd-bus
-              mountPath: /run/systemd/private
+              mountPath: /run/systemd
```

Comment:
We also need to apply the same changes to `deploy/kubernetes/base/node-daemonset.yaml` for Kustomization deployments